src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -41,7 +46,7 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
               artist {
                 __id
                 related {
-                  suggested(first: 1, exclude_followed_artists: true) {
+                  suggested(first: 3, exclude_followed_artists: true) {
```

Comment:
My idea is that if in a callback we'll fetch 3 suggestions at least one of those would be not suggested yet.

Reply:
We have first class support for an `exclude_artist_ids ` param to the `suggested` schema here, so we should basically be keeping track and passing this around (maybe using React context, maybe can just use props and `onFollow`, etc)

Reply:
👍 Did not know about that. Will change. Thanks!